src/QirRuntime/README.md

```diff
@@ -1,6 +1,7 @@
 # The Native QIR Runtime
 
-This folder contains QIR runtime project, which includes implementation of the
+This folder contains the Quantum Intermediate Representation (QIR) Runtime project. QIR is a subset of the [LLVM](https://llvm.org/) Intermediate Representation.
```

Comment:
You've used the definite article "the" to refer to LLVM IR, but not to QIR; I'd suggest either adding a "the" before "QIR" or dropping the "the" before "LLVM."